tests/png-visualizer.toit

```diff
@@ -180,14 +197,14 @@ abstract class PngVisualizingDriver_ extends AbstractDriver:
         pixels[y] |= 0b01010101
         pixels[y + patch_width - 1] |= 0b01010101
 
-  draw_byte_outline_ outline/int pixels/ByteArray patch_width/int -> none:
+  draw_byte_outline_ outline/int pixels/ByteArray patch_width/int --dotted=false -> none:
     bottom_left := pixels.size - patch_width
     // Dotted line along top and bottom.
-    for x := 0; x < patch_width; x += 2:
+    for x := 0; x < patch_width; x += dotted ? 2 : 1:
```

Comment:
I would create a local `step` outside.
It's used twice (here and below), and makes it (hopefully) easier to read.